From 6241641b2dbc7655e739af7002867d872d379377 Mon Sep 17 00:00:00 2001
From: Matt Wolenetz <wolenetz@chromium.org>
Date: Wed, 15 May 2013 12:32:23 -0700
Subject: [PATCH] Preserve M27 AVCodecIDs

This only works downstream, probably only for M29 roll, as
ffmpeg hardcodes checks into cmdutils.c that verify some
codec IDs versus a baseline that was reset after M27 and
after moving the AVCODEC_ID_SNOW to be an explicit MKBETAG
value.
See https://ffmpeg.org/trac/ffmpeg/ticket/2575
See also http://crbug.com/241212
---
 libavcodec/avcodec.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libavcodec/avcodec.h b/libavcodec/avcodec.h
index 8295f35..c6ad077 100644
--- a/libavcodec/avcodec.h
+++ b/libavcodec/avcodec.h
@@ -156,6 +156,7 @@ enum AVCodecID {
     AV_CODEC_ID_MSZH,
     AV_CODEC_ID_ZLIB,
     AV_CODEC_ID_QTRLE,
+    AV_CODEC_ID_SNOW_DEPRECATED,
     AV_CODEC_ID_TSCC,
     AV_CODEC_ID_ULTI,
     AV_CODEC_ID_QDRAW,
-- 
1.8.2.1

